src/index.ts

```diff
@@ -1,4 +1,4 @@
-export { AbstractPool } from './pools/abstract-pool'
+// export { AbstractPool } from './pools/abstract-pool'
```

Comment:
Remove also the comment and we are ready :D

Reply:
😏

Reply:
Done https://github.com/pioardi/poolifier/pull/116/commits/e82a18ed6f1b871eeb9dc9df254cfad4508b7f2b